src/lib.rs

```diff
@@ -59,12 +59,14 @@ impl Bencher {
     ///
     /// # Timing loop
     ///
-    /// ``` ignore
-    /// start = Instant::new();
+    /// ```rust,no_run
+    /// # fn routine() {}
+    /// let iters = 4_000_000;
+    /// let start = std::time::Instant::now();
```

Comment:
I think it would be better to import `std::time::Instant` to avoid the full path here.